examples/pos_tagging/src/metric_handler.py

```diff
@@ -39,3 +41,13 @@ def postprocess(self, predictions, references):
             actual_predictions.append(actual_prediction)
             actual_labels.append(actual_label)
         return actual_predictions, actual_labels
+
+    def postprocess(self, score: Dict) -> Dict:
```

Comment:
This does not replicate the previous behavior, rather it operates like the `return_entity_level_metrics` set to True. Can we add a `return_entity_level_metrics: bool = True` parameter to the constructor just like in the previous implementation?

Reply:
I assumed that it is only implemented for seqeval metric (current implementation on `main`) you want it to be applicable to all metrics ?

Reply:
Let's move this class to trapper with the change I recommended as well as renaming it to SeqevalMetricHandler.